x-pack/plugins/global_search_bar/public/search_syntax/parse_search_params.test.ts

```diff
@@ -76,4 +76,18 @@ describe('parseSearchParams', () => {
       },
     });
   });
+
+  it('converts multiword searchable types to phrases so they get picked up as types', () => {
+    const mockSearchableMultiwordTypes = ['canvas-workpad', 'enterprise search'];
+    const searchParams = parseSearchParams(
+      'type:canvas workpad types:canvas-workpad hello type:enterprise search type:not multiword',
```

Comment:
Can we add `type:"canvas workpad"` to the list so we make sure it handles types with existing quotes?

Also by doing so, I realized that we pass twice the same types in the array, could we avoid dedupes when returning the array?  (I know it's not part of fixing the issue but we could use the momentum 😊). Thanks 👍

Reply:
Right, I didn't think about checking for already existing quotes. It should work but I'll cover it in the test. 

As for deduping, I wasn't really sure if this is what we want (although it's makes logical sense) but sure, I'll add that.